### PR TITLE
Do content filling

### PR DESCRIPTION
**Pull request by Magons**
- [x] describe the OAuth2me installation 
- [x] replace all pictures <br> 
  dependency #1

### CODE REVIEW

**Comment by Magons** [comment]
@tehhi42 Look on [this](http://staging.oauth2me.divshot.io) pls.

**Comment by tehhi42** [comment]
@Magons checking

**Comment by tehhi42** [comment]
@Magons please see my comments about the design below:
- [x] Make 2 sections narrower.


https://jetthoughts.mybalsamiq.com/projects/oauth2me/New%20mockup


- [x]  Fix typo

**Comment by Magons** [comment]
@NazarKY pls [verify](http://staging.oauth2me.divshot.io/)

File: index.html [hunk 1]
```diff
@@ -68,33 +68,50 @@ <h3 class="centered">WHAT IS IT?</h3>
...
+                      <div class="highlight">
+                        <pre>$ bundle</pre>
+                      </div>
+                    </div>
+                    <div class="row testi">
+                      If you are using devise in your application, you should add this to your <span class="exhale">initializers/devise.rb</span>
+                    </div>
+                    <div class="row testi">
+                      <div class="highlight">
+                        <pre class="ruby" style="font-family:monospace;">OAuth2Me.<span style="color:#9900CC;">setup_devise</span>! <span style="color:#9966CC; font-weight:bold;">unless</span> Rails.<span style="color:#9900CC;">env</span>.<span style="color:#9900CC;">production</span>?</pre>
```
Comment (NazarKY) [inline index.html]: It's deprecate - we don't need this step anymore. Pls remove it

**Comment by Magons** [comment]
@tehhi42, @pftg  and @tmwh pls [verify](http://staging.oauth2me.divshot.io/) I walked away from the mockup. I think it's looks a little better. And I still don't know what write in "YOUR OWN OAUTH2ME ON GITHUB" section.

**Comment by pftg** [comment]
```
github: 'jetthoughts/oauth2-me-ruby'
```

I thought that we have released some version on rubygems

**Comment by pftg** [comment]
@tehhi42 pls check http://note.io/1Lv7xBb

**Comment by pftg** [comment]
also about feature per branch is one of option, and not so popular. Other popular problem:
1. For each new projects you should go to on OAuth2 provider site and setup new app. So with this gem you do not need it at all.
2. You want to share your public app which use OAuth2 and you want to add `Deploy to Heroku` button in order to have one-click installation you should use our gem. So user should not setup their app on oauth2 providers in order to check how your app works.

**Comment by tehhi42** [comment]
@Magons delete the bracket, please. http://note.io/1Lv7xBb

**Comment by Magons** [comment]
@pftg @tehhi42  pls check [it](http://development.oauth2me.divshot.io/)

**Comment by pftg** [comment]
@Magons please squash commits

**Pull request merged**